components/CaseStatus/CaseStatusFlag/CaseStatusFlag.tsx

```
@@ -9,51 +8,44 @@ interface Props {
   person: Resident;
 }
 
-const CaseStatusFlag = ({ person }: Props): React.ReactElement => {
-  const { data, error } = useCaseStatuses(person.id);
+const CaseStatusFlag = ({ person }: Props): React.ReactElement | null => {
+  const { data } = useCaseStatuses(person.id);
 
-  if (error) {
```

we don't care about errors here, let it fail silently